assets/src/edit-story/components/canvas/gridview/index.js

```diff
@@ -58,36 +58,67 @@ const RangeInputWrapper = styled.div`
   margin: 0 auto 75px auto;
 `;
 
-const Rectangle = styled.div`
+const Rectangle = styled.button.attrs(({ isDisabled }) => ({
+  disabled: isDisabled,
+}))`
+  border: 0;
+  padding: 0;
+  margin: 0;
+  min-width: unset;
+  background: transparent;
+  height: auto;
   display: flex;
   justify-content: center;
   align-items: center;
-  font-size: 32px;
+  cursor: pointer;
   color: ${({ theme }) => theme.colors.fg.v1};
   ${({ isLarge }) => (isLarge ? 'margin-left' : 'margin-right')}: 20px;
 
+  &:active {
+    outline: none;
+  }
+
+  ${({ disabled }) =>
+    disabled &&
+    `
+		pointer-events: none;
+		opacity: .3;
+	`}
+
   svg {
     width: ${({ isLarge }) => (isLarge ? '20px' : '12px')};
     height: auto;
     shape-rendering: crispEdges; /* prevents issues with anti-aliasing */
   }
 `;
 
-function RangeControl({ value, onChange }) {
+function RangeControl({ value, onChange, min, max, step }) {
+  const updateRangeValue = (addition) => {
+    onChange(Math.min(max, Math.max(min, value + addition)));
+  };
+
   return (
     <RangeInputWrapper>
-      <Rectangle>
+      <Rectangle
+        onClick={() => updateRangeValue(-step)}
+        isDisabled={value === min}
+      >
         <RectangleIcon />
       </Rectangle>
       <RangeInput
-        min="1"
-        max="3"
-        step="1"
+        min={min}
+        max={max}
+        step={step}
         value={value}
         onChange={(evt) => onChange(Number(evt.target.value))}
         thumbSize={24}
+        flexGrow={1}
```

Comment:
This needs some extra accessibility. I recommend something like `aria-label="Thumbnail size" aria-valuetext="Small/medium/large"` (of course vary the last one with actual size and both must use i18n).

Reply:
Done